plugin/type_uuid/mysql-test/type_uuid/func_uuidv4_plugin.result

```diff
@@ -0,0 +1,35 @@
+#
+# Start of 11.5 tests
```

Comment:
Technically its going to make it into 11.6. The 11.5 preview is already out.

Don't worry about change github PR, but if you could edit the test to assume 11.6 that would be good.